sentry-sidekiq/lib/sentry/sidekiq/cron/job.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+return unless defined?(::Sidekiq::Cron::Job)
+
+module Sentry
+  module Sidekiq
+    module Cron
+      module Job
+        def save
+          # validation failed, do nothing
+          return false unless super
+
+          # fail gracefully if can't find class
+          klass_const =
+            begin
+              ::Sidekiq::Cron::Support.constantize(klass.to_s)
+            rescue NameError
+              return true
+            end
+
+          # only patch if not explicitly included in job by user
+          unless klass_const.send(:ancestors).include?(Sentry::Cron::MonitorCheckIns)
+            klass_const.send(:include, Sentry::Cron::MonitorCheckIns)
+            klass_const.send(:sentry_monitor_check_ins,
+                             slug: name,
+                             monitor_config: Sentry::Cron::MonitorConfig.from_crontab(cron))
+          end
```

Comment:
Should we return `true` in the end?

Reply:
the upstream method doesn't return true either, just the falsey return is important.

Reply:
But the upstream's last line is [`Sidekiq.logger.info`](https://github.com/sidekiq-cron/sidekiq-cron/blob/master/lib/sidekiq/cron/job.rb#L481), which returns `true`

```
> ::Sidekiq.logger.info(123)
=> true
```

And `unless ...` returns `nil`. So this patch actually changes the code's behaviour here:

```
errors[job.name] = job.errors unless job.save
```